apps/jsonrpc2/test/jsonrpc2/http_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule JSONRPC2.HTTPTest do
+  use ExUnit.Case, async: false
+  alias JSONRPC2.Clients.HTTP
+  alias JSONRPC2.Servers.WebSocketHTTP
+  alias JSONRPC2.SpecHandlerTest
+
+  setup_all do
+    port = :rand.uniform(65_535 - 1025) + 1025
```

Comment:
This is basically ensuring the port is between `1026` and `65_535`, correct?

Reply:
yeah